tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -174,6 +209,99 @@ void doTransform(const Eigen::Affine3d& t_in,
   t_out = Eigen::Affine3d(transformToEigen(transform) * t_in);
 }
 
+template <>
+inline
+void doTransform(const Eigen::Isometry3d& t_in,
+                 Eigen::Isometry3d& t_out,
+                 const geometry_msgs::msg::TransformStamped& transform) {
+  t_out = Eigen::Isometry3d(transformToEigen(transform) * t_in);
+}
+
+/** \brief Convert a Eigen Quaterniond type to a Quaternion message.
+ * This function is a specialization of the toMsg template defined in tf2/convert.h.
+ * \param in The Eigen Quaterniond to convert.
+ * \return The quaternion converted to a Quaterion message.
+ */
+inline
+geometry_msgs::msg::Quaternion toMsg(const Eigen::Quaterniond& in) {
+ geometry_msgs::msg::Quaternion msg;
```

Comment:
There is only one space of indent. I think ROS style requires two.

Reply:
+1

Reply:
390955226773e4de5a6143338e632425071d896c